pkg/sampler/cache.go

```diff
@@ -323,6 +387,9 @@ func (s *sampleCache) metricToSample(
 	return sample
 }
 
+// cgroups resource file was not updated between two consecutive fetches.
```

Comment:
I'm not sure I understand this comment.

Reply:
I don't recognize this comment. It shouldn't be part of this PR, I will double-check.

Reply:
Ah, yes. Removed.